test/extended/scheduling/dra.go

```diff
@@ -755,5 +756,74 @@ var _ = g.Describe("[sig-scheduling][OCPFeatureGate:DynamicResourceAllocation]",
 				"v1 should be the preferred version")
 		})
 
+		// Adding a test for the DynamicResourceAllocation field in Scheduler ProfileCustomizations that depends on downstream DynamicResourceAllocation feature gate.
+		g.It("should support DynamicResourceAllocation field in Scheduler ProfileCustomizations [apigroup:config.openshift.io]", func(ctx context.Context) {
```

Comment:
not sure what value this test provides specific to DRA, if we want to validate that setting this value to either `true` or `false` enables/disables the dra plugin in the scheduler, then the test needs to wait for the scheduler to rollout and then validates appropriately.

Reply:
Agreed